### PR TITLE
[CBRD-25087] Restart cub_javasp via signal if JNI crashes the process

### PR DESCRIPTION
**Pull request by hgryoo**
http://jira.cubrid.org/browse/CBRD-25087

- Remove `-Xrs` option to enable signal handling from JVM.
- implement `javasp_signal_handler()` 
-- restart cub_javasp process if JVM is crashed
-- write ER_SP_SERVER_CRASHED error message in <db_name>_java.err

### CODE REVIEW

File: msg/ko_KR.utf8/cubrid.msg [hunk 1]
```diff
@@ -1443,7 +1443,11 @@ $ LOADDB
 1357 SQL 문자열을 와이드 문자열로 변환 실패.
 1358 영향을 받은 행 수를 알 수 없음.
 
-1359 마지막 에러
+1359 Java VM crashed: %1$s
+
+1360 Java VM 에 장애가 발생함. : %1$s
```
Comment (mhoh3963) [inline msg/ko_KR.utf8/cubrid.msg]: Unlike msg files in other languages, there are two 1360. Please confirm it.

Comment (hgryoo) [reply msg/ko_KR.utf8/cubrid.msg]: fixed

File: src/executables/javasp.cpp [hunk 2]
```diff
@@ -282,6 +305,82 @@ main (int argc, char *argv[])
...
+      for (int i = 0; i < nn_addresses; i++)
+	{
+	  err_msg += symbols[i];
+	  if (i < nn_addresses - 1)
+	    {
+	      err_msg += "\n";
+	    }
+	}
+      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_SP_SERVER_CRASHED, 1, err_msg.c_str ());
+
```
Comment (mhoh3963) [inline src/executables/javasp.cpp:364]: It would be a good idea to free the symbols allocated memory by bracktrace_symbol(), but you don't have to.

Comment (hgryoo) [reply src/executables/javasp.cpp:364]: Thank you.
```suggestion
      free(symbols);
      er_set (ER_ERROR_SEVERITY, ARG_FILE_LINE, ER_SP_SERVER_CRASHED, 1, err_msg.c_str ());
```

**Review by beyondykk9 — APPROVED** [review]

**Review by mhoh3963 — APPROVED** [review]

**Pull request merged**